Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5046.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if APP
```

Comment:
Why the `APP` directive?

Reply:
I copied the structure from Issue5268.xaml.cs so I thought the APP directive was in some kind neccessary for something :).  
In the other Issue classes the methods InitializeComponent and the click handlers are all wrapped in `#if APP` and if I don't do that I get some Intellisense errors (but no compiler errors).  
So I changed it now so that only InitializeComponent and Button_Clicked is wrapped in `#if APP`.

Reply:
Strange, I haven't seen this one before so I'm not sure what it does. But I don't think it's necessary for this.